v19.1/debug-encryption-active-key.md

```diff
@@ -0,0 +1,35 @@
+---
+title: View the encryption algorithm and store key
+summary: Learn the command for viewing the algorithm and store key for an encrypted store.
+toc: true
+---
+
+The `debug encryption-active-key` [command](cockroach-commands.html) displays the encryption algorithm and store key for an encrypted store.
+
+## Synopsis
+
+~~~ shell
+$ cockroach debug encryption-active-key [path specified by the store flag]
+~~~
+
+## Example
+
+Start a node with encryption-at-rest enabled:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start --store=cockroach-data --enterprise-encryption=path=cockroach-data,key=aes-128.key,old-key=plain --insecure --certs-dir=certs
+~~~
+
+View the encryption algorithm and store key:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach debug encryption-active-key cockroach-data
```

Comment:
Do we want a sample output? If so, here's one:
```
$ cockroach debug encryption-active-key cockroach-data
AES128_CTR:be235c29239aa84a48e5e1874d76aebf7fb3c1bdc438cec2eb98de82f06a57a0
```

Reply:
Hehe.. just asked you if we want to document the command :)

Reply:
Added the output.